src/components/dashboard/Marketplace.native.js

```diff
@@ -1,3 +1,113 @@
 // TODO: RN
+import React, { useEffect, useMemo, useState } from 'react'
```

Comment:
this file looks the same to me as web, why do we need .native?
The only thing to take care of here is to make sure the isIOS parts are only executed for mobile web for native it should have same behaviour as on web (ie opening a webview)

Reply:
We agree, I added a solution for this, reuse function createIframe.